content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,143 @@
-===================================
-Connect an IoT Box to your database
-===================================
+=======================================
+Connect an IoT Box to the Odoo database
+=======================================
 
-Install the Internet of Things (IoT) App on your Odoo Database.
+Install the :menuselection:`Internet of Things (IoT)` App on the Odoo database.
 
 .. image:: connect/connect01.png
    :align: center
+   :alt: The Internet of Things (IoT) App on the Odoo database.
 
-Go in the IoT App and click on Connect on the IoT Boxes page.
+Go to the :menuselection:`IoT` App and click on :guilabel:`Connect`` on the
+:abbr:`IoT (Internet of Things)` IoT Boxes page.
 
-.. image:: connect/connect02.png
+.. image:: connect/connect09.png
    :align: center
+   :alt: Connecting an IoT box to the Odoo database.
 
-Follow the steps to connect your IoT Box.
+Follow the steps to connect the IoT Box either via wired ethernet connection or via WiFi.
 
-.. image:: connect/connect03.png
+.. image:: connect/connect-iot-box.png
    :align: center
+   :alt: Connection steps for a wired connection or WiFi connection.
+
+.. note::
+   Ensure the :abbr:`IoT (Internet of Things)` box is flashed with the most up-to-date
+   :abbr:`IoT (Internet of Things)` image.
 
 Ethernet Connection
 ===================
 
-#. Connect to the IoT Box all the devices that have to be connected with
-   cables (ethernet, usb devices, etc.).
+#. Connect all the devices that have to be connected with cables to the
+   :abbr:`IoT (Internet of Things)` box (ethernet, usb devices, etc.).
 
-#. Power on the IoT Box.
+#. Power on the :abbr:`IoT (Internet of Things)` Box.
 
-#. Read the Pairing Code from a screen or a receipt printer connected to the IoT Box.
+#. Read the :guilabel:`Pairing Code` from a screen or a receipt printer connected to the
+   :abbr:`IoT (Internet of Things)` Box.
 
-   .. image:: connect/connect04.png
+   .. note::
+      If no screen is attached to the :abbr:`IoT (Internet of Things)` box then the pairing code can
+      be accessed from the :abbr:`IoT (Internet of Things)` box admin console by clicking on
+      :guilabel:`POS Display`. For how to access the admin console visit
+      :ref:`iot_connect/token`
 
-#. Input the Pairing Code and click on the Pair button.
+#. Input the :guilabel:`Pairing Code` on the :abbr:`IoT (Internet of Things)` module of the database
```

Comment:
"module" -> technically it is an app not a module. Except if it is for style purposes, I think it would be better (and less ambiguous) to keep the "app"

Reply:
Noted, I've searched for other instances that also refer to the "module", and replaced all. Thank you.